### PR TITLE
Add export_project option

### PR DESCRIPTION
**Pull request by terrymyc**
The new API export_project may create a `~/.asreview/tmp` folder to store the zip file.

### CODE REVIEW

File: asreview/webapp/src/Components/ExportDialog.js [hunk 1]
```diff
@@ -86,7 +108,7 @@ const ExportDialog = (props) => {
         aria-labelledby="scroll-dialog-title"
         aria-describedby="scroll-dialog-description"
       >
-        <DialogTitle id="scroll-dialog-title">Export results {props.project_id}</DialogTitle>
+        <DialogTitle id="scroll-dialog-title">Export results/project {props.project_id}</DialogTitle>
```
Comment (J535D165) [inline asreview/webapp/src/Components/ExportDialog.js:111]: @MartijnUX We should think about the title of this screen (or maybe the dual functionality of this dialog at all.)

**Comment by terrymyc** [comment]
image

File: asreview/webapp/src/Components/ExportDialog.js [hunk 2]
```diff
@@ -76,6 +76,28 @@ const ExportDialog = (props) => {
 
   }
 
+  const downloadProject = () => {
+
+    const project_id = store.getState()["project_id"]
+
+    if (project_id !== null){
```
[the remark this answers is not in the document]
Comment (terrymyc) [reply asreview/webapp/src/Components/ExportDialog.js:83]: Then nothing happens while clicking on the button...

**Comment by J535D165** [comment]
Thanks a lot, very nice addition!

**Pull request merged**